src/rgw/rgw_rest_s3.cc

```diff
@@ -3063,6 +3095,8 @@ RGWOp *RGWHandler_REST_Obj_S3::op_get()
     return new RGWGetACLs_ObjStore_S3;
   } else if (s->info.args.exists("uploadId")) {
     return new RGWListMultipart_ObjStore_S3;
+  } else if (is_tagging_op()) {
+    return new RGWGetObjTags_ObjStore_S3;
```

Comment:
@chardan it's handled, as it were, though we'd all be happier if we had a different stategy for handler setup that avoided allocation